generate_testdata.sh

```diff
@@ -64,6 +64,7 @@ scaffold_test_project() {
             $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
             $kb create webhook --group crew --version v1 --kind FirstMate --conversion
             $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
+	    $kb create webhook --group crew --version v1 --kind Admiral --defaulting
```

Comment:
nit space to be alined with the other lines.

Reply:
Done. There was glitch in my editor, it was showing fine there. My bad :)